Cargo.toml

```diff
@@ -145,4 +145,4 @@ image = "0.13"
 winit = "0.6"
 
 [target.x86_64-unknown-linux-gnu.dev-dependencies]
-glfw = "0.12"
+glfw = "0.14"
```

Comment:
Actually, I don't know why it's even here? gfx_app doesn't use glfw or SDL anyway.
Let's remove it?